cranelift/filetests/filetests/isa/x86/windows_fastcall_x64.clif

```diff
@@ -32,6 +32,31 @@ block0(v0: i64, v1: i64, v2: i64, v3: i64, v4: i64):
 }
 ; check: function %five_args(i64 [%rcx], i64 [%rdx], i64 [%r8], i64 [%r9], i64 [32], i64 fp [%rbp]) -> i64 fp [%rbp] windows_fastcall {
 
+; check that we preserve xmm6 and above if we're using them locally
+function %float_callee_saves(f64, f64, f64, f64) windows_fastcall {
+block0(v0: f64, v1: f64, v2: f64, v3: f64):
+; explicitly use a callee-save register
+[-, %xmm6]  v4 = fadd v0, v1
+[-, %xmm7]  v5 = fadd v0, v1
+    return
+}
+; check: function %float_callee_sav(f64 [%xmm0], f64 [%xmm1], f64 [%xmm2], f64 [%xmm3], i64 fp [%rbp], f64 csr [%xmm6], f64 csr [%xmm7]) -> i64 fp [%rbp], f64 csr [%xmm6], f64 csr [%xmm7] windows_fastcall {
+; nextln:                         ss0 = spill_slot 32, offset -80
+; nextln:                         ss1 = incoming_arg 16, offset -48
+; check: block0(v0: f64 [%xmm0], v1: f64 [%xmm1], v2: f64 [%xmm2], v3: f64 [%xmm3], v6: i64 [%rbp], v8: f64 [%xmm6], v9: f64 [%xmm7]):
+; nextln:                         x86_push v6
+; nextln:                         copy_special %rsp -> %rbp
+; nextln:                         adjust_sp_down_imm 64
+; nextln:                         v7 = stack_addr.i64 ss0
+; nextln:                         store notrap aligned v8, v7
+; nextln:                         store notrap aligned v9, v7+16
+; check:                          v10 = stack_addr.i64 ss0
+; nextln:                         v11 = load.f64 notrap aligned v10
```

Comment:
I had [a comment](https://github.com/bytecodealliance/cranelift/pull/1378#discussion_r376126112) in the original PR referencing a line I couldn't get to test properly here, but would make me happier these tests being robust in the face of the future.

The exact text doesn't match anymore but the same idea applies. I would like to test that preserves and restores have the right registers. In theory a `nextln` like
```
; nextln: [Mp2fld#710,%xmm6]                  v11 = load.f64 notrap aligned v10
```
would do the trick, but this yields a test error like
```
> [RexOp1spaddr8_id#808d,%rbp]        v10 = stack_addr.i64 ss0
                                      ^~~~~~~~~~~~~~~~~~~~~~~~
Matched #10: \bv10 = stack_addr\.i64 ss0\b
> [Mp2fld#710,%xmm6]                  v11 = load.f64 notrap aligned v10
Missed #11: \[Mp2fld\#710,%xmm6\]      v11 = load\.f64 notrap aligned v10\b
```
which is suspicious because the only difference on the matching lines is whitespace. When whitespace is equalized, this still yields an error. We use specific checks like this [in other filetests](https://github.com/bytecodealliance/wasmtime/blob/f7c2a58d236bf23819139930bf1e469a47e08140/cranelift/filetests/filetests/isa/x86/probestack-disabled.clif#L18-L23) also run as `test compile`, so I suspect this is a bug in the filetest runner?

I would greatly appreciate thoughts if anyone is particularly familiar with `filetest`.